### PR TITLE
Add StreamMarketTrades API and custom config fields

### PR DESCRIPTION
**Pull request by shubhamgoyal42**
tested on local and hubblenext

### CODE REVIEW

File: plugin/evm/orderbook/contract_events_processor.go [hunk 1]
```diff
@@ -542,3 +589,8 @@ func (cep *ContractEventsProcessor) updateMetrics(logs []*types.Log) {
 	ordersPlacedPerBlock.Update(orderPlacedCount)
 	ordersCancelledPerBlock.Update(orderCancelledCount)
 }
+
+func getAddressFromTopicHash(topicHash common.Hash) common.Address {
+	address32 := topicHash.String() // address in 32 bytes with 0 padding
+	return common.HexToAddress(address32[:2] + address32[26:])
```
Comment (atvanguard) [inline plugin/evm/orderbook/contract_events_processor.go:595]: use `return common.BytesToAddress(topicHash.Bytes())`

File: plugin/evm/orderbook/contract_events_processor.go [hunk 2]
```diff
@@ -484,24 +499,56 @@ func (cep *ContractEventsProcessor) PushtoTraderFeed(events []*types.Log, blockS
 		timestamp, _ := args["timestamp"]
```
Comment (atvanguard) [inline plugin/evm/orderbook/contract_events_processor.go:499]: `unnecessary assignment to the blank identifier (S1005)go-staticcheck`

File: plugin/evm/orderbook/contract_events_processor.go [hunk 3]
```diff
@@ -484,24 +499,56 @@ func (cep *ContractEventsProcessor) PushtoTraderFeed(events []*types.Log, blockS
...
 }
 
-func getAddressFromTopicHash(topicHash common.Hash) common.Address {
-	address32 := topicHash.String() // address in 32 bytes with 0 padding
-	return common.HexToAddress(address32[:2] + address32[26:])
+func (cep *ContractEventsProcessor) PushToMarketFeed(events []*types.Log, blockStatus BlockConfirmationLevel) {
+	for _, event := range events {
+		args := map[string]interface{}{}
+		switch event.Topics[0] {
+		case cep.clearingHouseABI.Events["PositionModified"].ID:
```
[the remark this answers is not in the document]
Comment (shubhamgoyal42) [reply plugin/evm/orderbook/contract_events_processor.go:522]: PositionModified should give them all the details they need about trades

File: plugin/evm/orderbook/contract_events_processor.go (hunk 2 above)
Comment (shubhamgoyal42) [reply plugin/evm/orderbook/contract_events_processor.go]: fixed

File: plugin/evm/orderbook/contract_events_processor.go (hunk 1 above)
Comment (shubhamgoyal42) [reply plugin/evm/orderbook/contract_events_processor.go]: fixed

File: networks/hubblenext/chain.json [hunk 4]
```diff
@@ -7,5 +7,7 @@
   "priority-regossip-txs-per-address": 20,
   "priority-regossip-addresses": ["0xeAA6AE79bD3d042644D91edD786E4ed3d783Ca2d"],
   "validator-private-key-file": "/home/ubuntu/validator.pk",
-  "feeRecipient": "0x393bd9ac9dbBe75e84db739Bb15d22cA86D26696"
+  "feeRecipient": "0x393bd9ac9dbBe75e84db739Bb15d22cA86D26696",
+  "is-validator": true,
+  "trading-api-enabled": true
```
Comment (atvanguard) [inline networks/hubblenext/chain.json:12]: this can be moved to `chain_api_node.json`

Comment (shubhamgoyal42) [reply networks/hubblenext/chain.json]: done

**Review by atvanguard — APPROVED** [review]

**Pull request merged**